### PR TITLE
[OpenGL] Fix lib files copy when VCPKG_BUILD_TYPE is set

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@AenBleidd, thanks for the PR!

Could you also update Version to  0.0-6 in CONTROL file?

**Comment by AenBleidd** [comment]
@PhoebeHui, done. Thanks for the fast review!

File: ports/opengl/portfile.cmake [hunk 1]
```diff
@@ -24,22 +24,32 @@ if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore
...
-        ${CURRENT_PACKAGES_DIR}/lib
-        ${CURRENT_PACKAGES_DIR}/debug/lib
-        ${CURRENT_PACKAGES_DIR}/share/opengl
-    )
+    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL release)
+        file(MAKE_DIRECTORY
+            ${CURRENT_PACKAGES_DIR}/include/gl
+            ${CURRENT_PACKAGES_DIR}/lib
+            ${CURRENT_PACKAGES_DIR}/share/opengl
+        )
```
Comment (Neumann-A) [inline ports/opengl/portfile.cmake:32]:
```
${CURRENT_PACKAGES_DIR}/include/gl
and
${CURRENT_PACKAGES_DIR}/share/opengl
```
are required in a debug only build

Comment (AenBleidd) [reply ports/opengl/portfile.cmake]: Fixed

**Review by PhoebeHui — APPROVED** [review]

File: ports/opengl/portfile.cmake [hunk 2]
```diff
@@ -26,20 +26,32 @@ if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore
 
     file(MAKE_DIRECTORY
         ${CURRENT_PACKAGES_DIR}/include/gl
-        ${CURRENT_PACKAGES_DIR}/lib
-        ${CURRENT_PACKAGES_DIR}/debug/lib
         ${CURRENT_PACKAGES_DIR}/share/opengl
     )
+    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL release)
```
Comment (strega-nil) [inline ports/opengl/portfile.cmake:31]:
```suggestion
    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
```

gotta make sure not to dereference the `release` variable :)

File: ports/opengl/portfile.cmake [hunk 3]
```diff
@@ -26,20 +26,32 @@ if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore
...
+        file(MAKE_DIRECTORY
+            ${CURRENT_PACKAGES_DIR}/lib
+        )
+    endif()
+    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL debug)
```
Comment (strega-nil) [inline ports/opengl/portfile.cmake:36]:
```suggestion
    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
```

File: ports/opengl/portfile.cmake [hunk 4]
```diff
@@ -26,20 +26,32 @@ if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore
...
     file(COPY
         "${HEADERSPATH}\\gl\\GL.h"
         "${HEADERSPATH}\\gl\\GLU.h"
         DESTINATION ${CURRENT_PACKAGES_DIR}/include/gl
     )
-    file(COPY ${LIBGLFILEPATH}  DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
-    file(COPY ${LIBGLUFILEPATH} DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
-    file(COPY ${LIBGLFILEPATH}  DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib)
-    file(COPY ${LIBGLUFILEPATH} DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib)
+    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL release)
```
Comment (strega-nil) [inline ports/opengl/portfile.cmake:47]: Same as above :)

```suggestion
    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
```

File: ports/opengl/portfile.cmake [hunk 5]
```diff
@@ -26,20 +26,32 @@ if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore
...
+        file(COPY ${LIBGLFILEPATH}  DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
+        file(COPY ${LIBGLUFILEPATH} DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
+    endif()
+    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL debug)
```
Comment (strega-nil) [inline ports/opengl/portfile.cmake:51]:
```suggestion
    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
```

**Review by strega-nil — APPROVED** [review]
Some minor changes, but this looks great! Thanks @AenBleidd :)

**Comment by strega-nil** [comment]
Oops, forgot to merge after I pushed changes; thanks @AenBleidd :D

**Comment by AenBleidd** [comment]
@strega-nil, thank you too :)

**Pull request merged**